### PR TITLE
feat/utils

### LINKED ISSUE #2: Create Rocket initial project structure
We need to have the initial Rocket framework setup in place, we can do so by having the following endpoints in the project:

```
GET /health

response:
200 OK { "status": "ok" }
```

```
POST /echo
request:
text

response:
200 OK incoming text
```

We can use Rocket's Getting started documentation as a basis for this work
https://rocket.rs/v0.5-rc/guide/getting-started/

### PR DESCRIPTION
**Pull request by lazoliver**
Closes #2

### CODE REVIEW

File: .vscode/settings.json [hunk 1]
```diff
@@ -0,0 +1,7 @@
+{
```
Comment (andref-dev) [inline .vscode/settings.json]: Let's add this folder (`.vscode`) to our gitignore. 

We can use this as reference:
https://stackoverflow.com/a/7927283

File: src/handlers/utils.rs [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+pub struct HealthResponse {
+    status: String
+}
+
+#[get("/health")]
+pub fn health_handler() -> Json<HealthResponse> {
+    let status = String::from("ok");
+    let health_response = HealthResponse {status};
+
+    Json(health_response)
```
Comment (andref-dev) [inline src/handlers/utils.rs:15]: 🔥

**Review by andref-dev — CHANGES_REQUESTED** [review]
I'm having the following error when trying to run the project:
```
error: failed to run custom build command for `pear_codegen v0.1.5`

Caused by:
  process didn't exit successfully: `/Users/andresantos/Projects/guesser-game-api/target/debug/build/pear_codegen-fb2bd6f806307bf0/build-script-build` (exit status: 101)
  --- stderr
  Error: Pear requires a 'dev' or 'nightly' version of rustc.
  Installed version: 1.75.0 (2023-12-21)
  Minimum required:  1.31.0-nightly (2018-10-05)
  thread 'main' panicked at /Users/andresantos/.cargo/registry/src/index.crates.io-6f17d22bba15001f/pear_codegen-0.1.5/build.rs:24:13:
  Aborting compilation due to incompatible compiler.
  note: run with `RUST_BACKTRACE=1` environment variable to display a backtrace
warning: build failed, waiting for other jobs to finish...

```

I understand what it means, my question is if we want to rely our project on non stable rust versions, should we?
If not, is there a way to avoid having this requirement?

let me know your thoughts, and if we're going with the nightly requirement, let's make sure to add that to our README later on

File: .vscode/settings.json (hunk 1 above)
Comment (lazoliver) [reply .vscode/settings.json]: Removed

**Comment by lazoliver** [comment]
First of all, i'm sorry, you can solve this problem using following commands bellow:
```terminal
rustup toolchain install nightly
rustup default nightly
```

**Review by andref-dev — APPROVED** [review]
LGTM 🚀 

One suggestion based on your last comment, isntead of changing my whole environment to be nightly, I changed just this project  using:
```
cd guesser-game-api
rustup override set nightly
```

**Pull request merged**